index.rst

```diff
@@ -486,10 +489,12 @@ Deliverable 4: Required Non-Scalar Metrics
 
    The deliverable description from the `charge`_ has been directly copied here to ease readability.
 
-  1. (`SITCOM-180`_) Provide a list of required non-scalar metrics are required and cannot be computed with analysis_tools.
+  4. (`SITCOM-180`_) Provide a list of required non-scalar metrics are required and cannot be computed with analysis_tools.
```

Comment:
Some extra words. Suggest "Provide a list of required non-scalar metrics that cannot be computed with analysis_tools."